_specs/ecip-1000.md

```diff
@@ -353,6 +353,12 @@ In addition, it is recommended that literal code included in the ECIP be dual-li
 * Some ECIPs, especially consensus layer, may include literal code in the ECIP itself which may not be available under the exact license terms of the ECIP.
 * Despite this, not all software licenses would be acceptable for content included in ECIPs.
 
+# Appendix: Irregular Process Transitions
+
+As a decentralized community, it is important that for all governance actions, especially in ECIP, the established process is followed. In this section, we document irregular process transitions, and give special permissions to them, to state that they followed the process. Note that this is essentially similar to theDAO hard fork in Ethereum, and should only be added in extreme cases.
+
+* In [PR#199](https://github.com/ethereumclassic/ECIPs/pull/199), Afri Schoedon merged a PR moving ECIP-1061 to "Last Call" status. At the time, Afri dismissed a review of clear objections from ECIP editor Wei Tang, without waiting for consultations of the community, or resolutions with the review submitter. We here declare that this particular action from Afri followed the ECIP process. 
```

Comment:
```suggestion
* In [PR#199](https://github.com/ethereumclassic/ECIPs/pull/199), @soc1c merged a PR moving ECIP-1061 to "Last Call" status. At the time, @soc1c dismissed a review of clear objections from ECIP editor Wei Tang, without waiting for consultations of the community, or resolutions with the review submitter. We here declare that this particular action from @soc1c followed the ECIP process. 
```

Reply:
Not sure this is helpful. Wei misunderstood the meeting and mentioned that in a blocking pull request review. I dismissed that after revisions of the document and two other ECIP editors approved the pull request. It all can be followed by reading through #199. Changing this sentence from "Afri" to "soc1c" does not resolve this in any way.

Reply:
Let's keep it as "Afri" then, since @soc1c mentioned that @BelfordZ's suggestion is not applicable.

Thanks Afri for your personal confirmation that you indeed dismissed the review, and thus this is indeed an irregular process transition.